src/nxdoc/nuxeo-server/platform-services/preview.md

```diff
@@ -190,6 +190,29 @@ if (mtPreviewer != null) {
 }
 ```
 
+### ZipFile Handling Details
+
+When previewing a zip, the files it contains are listed. The displayed filenames are decoded as `UTF-8`. The following exception can be raised at `ZipFile`s instanciation if the underlying zip was encoded in a charset different from `UTF-8` containing non-ASCII characters:
+
+```java
+ZipException: invalid CEN header (bad entry name)
+```
+> Character set detection is at best an imprecise operation.
+> <cite>Apache Tika</cite>
+
+Decoding such files can't be done by looping over a list of available charsets as some may work but give partial results (missing characters for example). To work around this, a fallback charset mechanism is provided to decode rebellious zips while keeping the whole platform working in `UTF-8`.
```

Comment:
This is not specific to zip files. 
By "such" I intend : "the underlying zip was encoded in a charset different from `UTF-8` containing non-ASCII characters" which happens to be a zip.
Otherwise we could simple state the following, which may just be less contextual ?
> Decoding ~zip~ files can't be done by looping over a list of available charsets.

Reply:
Thanks for clarifying, I think the less contextual option would be better in this case!